docs/source/app-dev/app-arch.rst

```diff
@@ -49,7 +49,7 @@ with a JSON API server by running
 
 in the root of the project. This is the most simple DAML ledger implementation. Once your
 application matures and becomes ready for production, the ``daml deploy`` command helps you deploy
-your frontend and DAML artefacts of your project to a production ledger. See :ref:`Deploying to DAML
+your frontend and DAML artifacts of your project to a production ledger. See :ref:`Deploying to DAML
```

Comment:
Interesting! Are "artifact" and "artefact" just the American and British spelling of the same word or do they actually mean slightly different things? Also, why is the "i" in "artifact" not pronounced like the "i" in "anti"? Given how "artifact" is pronounced, shouldn't it be spelled "artefact"? (I think I'll nominate this pair of words for our next language team meeting's section "Hard word of the week" 😄)

Reply:
yes, it's just US vs UK spelling. We try to consistently use US spelling.